src/compute_centroid_stress_atom.cpp

```diff
@@ -228,20 +256,18 @@ void ComputeCentroidStressAtom::compute_peratom()
 
   if (kspaceflag && force->kspace && force->kspace->compute_flag) {
     double **vatom = force->kspace->vatom;
-    for (i = 0; i < nkspace; i++) {
-      for (j = 0; j < 6; j++)
-        stress[i][j] += vatom[i][j];
-      for (j = 6; j < 9; j++)
-        stress[i][j] += vatom[i][j-3];
-    }
+    for (i = 0; i < nkspace; i++)
```

Comment:
@sjplimp since `kspace->cvatom` doesn't exist, it's unclear what your intentions are with this change.